addons/hr_holidays/models/hr_leave.py

```diff
@@ -971,9 +971,13 @@ def create(self, vals_list):
                     holiday_sudo.message_subscribe(partner_ids=holiday.employee_id.leave_manager_id.partner_id.ids)
                 if holiday.validation_type == 'no_validation':
                     # Automatic validation should be done in sudo, because user might not have the rights to do it by himself
+                    date_from = format_datetime(self.env, holiday.date_from) or ""
+                    date_to = format_datetime(self.env, holiday.date_to) or ""
                     holiday_sudo.action_validate()
                     holiday_sudo.message_subscribe(partner_ids=[holiday._get_responsible_for_approval().partner_id.id])
-                    holiday_sudo.message_post(body=_("The time off has been automatically approved"), subtype_xmlid="mail.mt_comment") # Message from OdooBot (sudo)
+                    holiday_sudo.message_post(
+                        body=_("The time off has been automatically approved") + f"<br /><br />\n{date_from} - {date_to}",
+                        subtype_xmlid="mail.mt_comment") #  Message from OdooBot (sudo)
```

Comment:
This is not needed. First of all, for no_validation - _get_responsible_for_approval() does not return anything, so, noone will be receiving this message. Moreover, even if someone was receiving, it's still not enough info - who's time off, what type? So, the person still needs to open the record. 

This can dropped.

Reply:
@sofiagvaladze 
Have you read and looked at the screenshots on the ticket? This is indeed needed as the manager does get that something was approved, but without any date...the subject does say who and what kind of time off was approved...so the date(time) is the important missing piece.

Reply:
and what will `_get_responsible_for_approval` return for 'no_validation' type?  --->  `self.env.user`. 

and `message_post` does not notify the  author of its own messages, thus, so noone will be receiving the mail

Reply:
Hmm, IMO, it does only return the user in case the time off mode is **not per employee** which is the standard use case when someone is creating a time off, right?

```python
     if self.holiday_type != 'employee': 
         return responsible 
```
So, normally someone else will be subscribed and will get the notification...but unrelated to this detail, why not give proper information about the time off via notication? 

https://github.com/odoo/odoo/blob/b57bc9ca8aa9d3290ef0cead477850c3ba61c86e/addons/hr_holidays/models/hr_leave.py#L1516-L1533

Reply:
Ok, wrong spot to proof that the manager is subscribed...it is above another if

https://github.com/odoo/odoo/blob/b57bc9ca8aa9d3290ef0cead477850c3ba61c86e/addons/hr_holidays/models/hr_leave.py#L970-L975

Reply:
for `validation_type` of `no_validation` it won't go into any of those if statements and  responsible won't be re-assigned.
For now, that text `The time off has been automatically approved` - served as an information in the chatter, that's it. 

I am guessing that you have customized `_get_responsible_for_approval` and are assigning someone and that's when that someone is getting the mail notification.  So, it's not an issue in standard